### ISSUE TITLE
Pypandoc fails to unpack pandoc deb file

### ISSUE THREAD

**Issue by juhtie01** [bebraw/pypandoc#146]
Looks like pypandoc 1.4 fails to unpack the latest pandoc deb file.

```tar (child): data.tar.gz: Cannot open: No such file or directory
tar (child): Error is not recoverable: exiting now
tar: Child returned status 2
tar: Error is not recoverable: exiting now
* Using already downloaded file pandoc-2.0.0.1-1-amd64.deb
* Unpacking pandoc-2.0.0.1-1-amd64.deb to tempfolder...
Traceback (most recent call last):
  File "exec_summary_report.py", line 10, in <module>
    download_pandoc()
  File "/home/jenkins/workspace/services_milestone_report/tools/executive_summary_report/.tox/docs/lib/python3.5/site-packages/pypandoc/pandoc_download.py", line 207, in download_pandoc
    unpack(filename, targetfolder)
  File "/home/jenkins/workspace/services_milestone_report/tools/executive_summary_report/.tox/docs/lib/python3.5/site-packages/pypandoc/pandoc_download.py", line 86, in _handle_linux
    subprocess.check_call(cmd)
  File "/usr/local/lib/python3.5/subprocess.py", line 271, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['tar', 'xzf', 'data.tar.gz']' returned non-zero exit status 2
```

Instead of `data.tar.gz`, the deb file now includes a `data.tar.xz` file.

**Comment by janschulz** [comment]
This commit has the needed code changes  https://github.com/conda-forge/pandoc-feedstock/commit/6e1dd08

**Comment by synap5e** [comment]
I use pypandoc as part of a CI script and was having this issue.

As a workaround I changed the line
`python -c "__import__('pypandoc').download_pandoc()"`
to 
`python -c "__import__('pypandoc').download_pandoc(version='1.19.1')"`

**Comment by jankatins** [comment]
Fixed in #154
